docs/design/usage-based-scheduling.md

```diff
@@ -57,9 +62,9 @@ tiers:
       - name: nodeorder
       - name: binpack
 metrics:                               # metrics server related configuration
-  type: prometheus                     # Optional, The metrics source type, prometheus by default, support prometheus and elasticsearch
+  type: prometheus                     # Optional, The metrics source type, prometheus by default, support "prometheus", "prometheus_adapt" and "elasticsearch"
   address: http://192.168.0.10:9090    # Mandatory, The metrics source address
-  interval: 30s                        # Optional, The scheduler pull metrics from Prometheus with this interval, 5s by default

```

Comment:
Is the default 30s too long?

Reply:
1. The Prometheus does not collect data frequently. Generally, the data collection frequency ranges from 10s to 60s. The frequency of obtaining monitoring data from the Prometheus does not need to be too frequent.
2. In a large-scale cluster (1000+ nodes) scenario, the Custom Metrics API is used to frequently obtain monitoring information, which causes great pressure on kube-apiserver.
3. Load awareness scheduling does not have strict requirements on real-time resources. Therefore, the indicator collection period can be prolonged.